src/akkudoktoreos/devices/battery.py

```diff
@@ -10,345 +10,273 @@
 logger = get_logger(__name__)
 
 
-def max_ladeleistung_w_field(default: Optional[float] = None) -> Optional[float]:
+def max_charging_power_field(description: Optional[str] = "") -> float:
     return Field(
```

Comment:
we should always provide an API description, maybe if it's None here, use a default description later

Reply:
added